.github/workflows/test.yaml

```diff
@@ -51,7 +51,7 @@ jobs:
 
     services:
       postgres:
-        image: postgres
+        image: postgres:15
```

Comment:
Why the bump ?

Reply:
The latest is 16 so I was actually trying to downgrade it. For some reason, migration failed with v16.
What's our preferred version?

Reply:
It looks like all major clouds now support 15, but if we bump it from 14 to 15 here, we also need to bump it everywhere else

Reply:
> but if we bump it from 14 to 15 here

I think this Github action has been running on **15** for quite some time now since it was the default.
But if we're running on 14 everywhere else, I'll tag this to 14 here as well.